qbx_consumables/server.lua

```diff
@@ -1,17 +1,49 @@
 local config = require 'qbx_consumables.config'
 
-local function addHunger(amount)
+--- sets player nutrition level in statebag
+--- @param source number
+--- @param amount number
+local function setHunger(source, amount)
     amount = lib.math.clamp(amount, 0, 100)
-    local playerState = Player(source).state
-    playerState:set('hunger', amount, true)
-    TriggerClientEvent('hud:client:UpdateNeeds', source, amount, playerState.thirst)
+    Player(source).state.hunger = amount
+
+    --- hotfix: remove when qbx_core issue #457 fixed
+    local player = exports.qbx_core:GetPlayer(source)
+    if not player then return end
+
+    player.Functions.SetMetaData('hunger', amount)
+    --- hotfix
+end
+
+--- raises player nutrition level in statebag
+--- @param source number
+--- @param amount number
+local function addHunger(source, amount)
+    local hunger = Player(source).state.hunger or 0
```

Comment:
How about “assert” when state.hunger = nil?

Reply:
Hmm, I have a couple concerns that I'm reviewing locally, give me a few and I'll get back to ya.
In the meantime, @Manason am I to assume by [qbx_core issue #456](https://github.com/Qbox-project/qbx_core/issues/456) that only OneSync Infinity is under scope of support?

Reply:
@ArturMichalak I've sent a PR to your fork that you can review.

Reply:
We only support OneSync Infinity yeah